src/ProofOfContribution.sol

```diff
@@ -152,14 +152,47 @@ abstract contract ProofOfContribution is ISoulbound, ERC1155, Ownable {
     //////////////////////////////////////////////////////////////*/
 
     /// @inheritdoc	ISoulbound
-    function issue(address _recipient, uint256 _tokenId) external onlyOwner {
+    function issue(address _recipient, uint256 _tokenId) external payable onlyOwner {
         _issue(_recipient, _tokenId);
+        
+        // note WORK IN PROGRESS
+
+        if (msg.value > 0) {
+            // forward any ether to recipient
+            (bool success, ) = _recipient.call{value: msg.value}("");
+            require(success, "ProofOfContribution: failed to send ether");
+        }
     }
 
     /// @inheritdoc	ISoulbound
-    function issueBatch(address[] calldata _recipients, uint256 _tokenId) external onlyOwner {
-        for (uint256 i = 0; i < _recipients.length; i++) {
-            _issue(_recipients[i], _tokenId);
+    function issueBatch(address[] calldata _recipients, uint256 _tokenId) external payable onlyOwner {
+        // note WORK IN PROGRESS
+
+        require(_recipients.length <= 100, "SBT: can not issue more than 100 SBTs in a single transaction");
+
+        if (msg.value > 0) {      
+            uint256 valueToSend = msg.value / _recipients.length;
+            bool success;
+
+            // issue SBTs and forward any ether to recipients, divided equally  
+            for (uint256 i = 0; i < _recipients.length; i++) {
+                _issue(_recipients[i], _tokenId);
+
+                (success, ) = _recipients[i].call{value: valueToSend}("");
+                require(success, "ProofOfContribution: failed to send ether");
+            }
+
+            // cleanup any dust leftover
+            uint256 balance = address(this).balance;
+            if (balance > 0) {
+                (success, ) = msg.sender.call{value: balance}("");
+                require(success, "ProofOfContribution: failed to send ether");
+            }
```

Comment:
There's a good chance this is guaranteed to be gas inefficient.  We should discuss